packages/cli/bin/dev

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env node
```

Comment:
Copied this file from new cli created via oclif. It works for any TS within /cli package.